2/twemoji.js

```diff
@@ -459,7 +464,11 @@ var twemoji = (function (
             ret = ret.concat(' ', attrname, '="', escapeHTML(attrib[attrname]), '"');
           }
         }
-        ret = ret.concat('>');
+        if (options.selfClosing) {
```

Comment:
I agree.  I've gotten strong push-back from other project maintainers when suggesting trivial application/xhtml+xml changes, so wanted to err on the side of caution and make it optional.  I'll kill this flag.